### PR TITLE
keys:*

### PR DESCRIPTION
**Pull request by dickeyxxx**

### CODE REVIEW

File: commands/keys/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  return `${key[0]} ${key[1].substr(0,10)}...${key[1].substr(-10,10)} ${cli.color.green(key[2])}`;
+}
+
+function* run(context, heroku) {
+  let keys = yield heroku.get('/account/keys');
+  if (context.flags.json) {
+    cli.log(JSON.stringify(keys, null, 2));
+  } else if (keys.length === 0) {
+    cli.warn('You have no ssh keys.');
+  } else if (context.flags.long) {
```
Comment (ransombriggs) [inline commands/keys/index.js]: @dickeyxxx Here and the block below you are missing the keys header
```
=== foo@bar.com Keys
```

Comment (dickeyxxx) [inline commands/keys/index.js]: updated

File: commands/keys/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+function* run(context, heroku) {
+  let keys = yield heroku.get('/account/keys');
+  if (context.flags.json) {
+    cli.styledJSON(keys);
+  } else if (keys.length === 0) {
+    cli.warn('You have no ssh keys.');
+  } else {
+    cli.styledHeader(`${cli.color.cyan(keys[0].email)} keys`);
```
Comment (ransombriggs) [inline commands/keys/index.js]: @dickeyxxx The authenticated user email should be printed here, as the key may have a different email associated with it.

Comment (ransombriggs) [inline commands/keys/index.js]: I stand corrected, that is the right email, I had assumed it was pulled from the key, but it appears I am was wrong.

File: commands/keys/remove.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+let co  = require('co');
+let cli = require('heroku-cli-util');
+
+function* run(context, heroku) {
+  yield cli.action(`Removing ${cli.color.cyan(context.args.key)} ssh key`, co(function* () {
+    let keys = yield heroku.get('/account/keys');
+    if (keys.length === 0) throw new Error('No ssh keys on account');
+    let key = keys.find(k => k.comment === context.args.key);
```
Comment (ransombriggs) [inline commands/keys/remove.js]: @dickeyxxx should handle the possibility that there are multiple matches here.

Comment (dickeyxxx) [inline commands/keys/remove.js]: updated

**Comment by ransombriggs** [comment]
@dickeyxxx lgtm

**Pull request merged**